ChildMindInstitute/mindlogger-admin-OLD-Vue
# Fix timezone in dashboard charts #347

Dates were being displayed incorrectly in the dashboard due to timezone. This was corrected in this PR.

Fixes #347

Testing this:
1. Create a new user response using the mobile app.
2. Log into the admin panel using the owner or reviewer account
3. Make sure that the new response is shown with today's date
